### PR TITLE
display latest posted chirp UI

### PR DESCRIPTION
**Pull request by jiabaow**
I add a simple component for showing the latest posted chirp
Screenshot 2021-11-17 at 22 07 07
(with hard-code data on Social page)
Same as the following one but solve snapshot fails problem by merging this to the master (on a new branch) 
https://github.com/dedis/student_21_pop/pull/592

### CODE REVIEW

File: fe1-web/parts/Social.tsx [hunk 1]
```diff
@@ -22,8 +25,31 @@ const styles = StyleSheet.create({
     width: 500,
     alignContent: 'flex-end',
   } as TextStyle,
+  item: {
+    backgroundColor: '#f9c2ff',
+    padding: 20,
+    marginVertical: 8,
+    marginHorizontal: 16,
+  },
```
Comment (Xelowak) [inline fe1-web/parts/Social.tsx:33]: Is it used anywhere ? It seems not.

**Review by Xelowak — APPROVED** [review]
That's a nice first component for chirps, thanks a lot ! We'll probably its design later

File: fe1-web/components/ChirpCard.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        </View>
+      </View>
+    </View>
+  );
+};
+
+const propTypes = {
+  sender: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+  time: PropTypes.instanceOf(Date).isRequired,
```
Comment (Xelowak) [inline fe1-web/components/ChirpCard.tsx:62]: Shouldn't it be like this ?
```suggestion
  time: PropTypes.instanceOf(Timestamp).isRequired,
```

Comment (pierluca) [reply fe1-web/components/ChirpCard.tsx:62]: I would agree.

**Review by pierluca — COMMENTED** [review]
Minor change, otherwise LGTM

**Review by pierluca — APPROVED** [review]

**Pull request merged**